abs/groonga/PKGBUILD

```
@@ -33,7 +33,7 @@ build() {
     --with-mecab \
     --enable-mruby
 
-    make
+    make -j$(nproc || PATH="/sbin:$PATH" sysctl -n hw.ncpu)
```

`sysctx ...` is needless because it's for *BSD such as FreeBSD and macOS:

```suggestion
    make -j$(nproc)
```

BTW, Groonga will remove GNU Autotools build system in the future. So it's better that we use CMake + Ninja instead of `make -jXXX` in near future.

---

Thanks @kou 

I was reading the wrong section when I wrote this🤦‍♂️

https://groonga.org/ja/docs/install/debian.html#build-from-source

`make -j$(nproc)` indeed👀